_plugins/generators/env_generator.rb

```diff
@@ -0,0 +1,12 @@
+
+module Jekyll
```

Comment:
Our env generation is currently handled by `jekyll-crds`

Reply:
Removing!